lib/flutter_chat_ui.dart

```diff
@@ -6,19 +6,27 @@ export 'src/models/bubble_rtl_alignment.dart';
 export 'src/models/emoji_enlargement_behavior.dart';
 export 'src/models/input_clear_mode.dart';
 export 'src/models/send_button_visibility_mode.dart';
+export 'src/models/typing_indicator_mode.dart';
 export 'src/util.dart' show formatBytes, isConsistsOfEmojis;
 export 'src/widgets/chat.dart';
 export 'src/widgets/chat_list.dart';
 export 'src/widgets/image_gallery.dart';
 export 'src/widgets/input/attachment_button.dart';
 export 'src/widgets/input/input.dart';
+export 'src/widgets/input/input.dart';
+export 'src/widgets/input/input_text_field_controller.dart';
```

Comment:
Hey export them twice? This has a happens a few times I. This file...

Reply:
Good point. I guess it has been missed during fixing merge conflicts. Will remove it.